samples/contrib/aws-samples/simple_train_pipeline/README.md

```diff
@@ -41,8 +41,31 @@ An example pipeline with only [train component](https://github.com/kubeflow/pipe
    boto3.resource('s3').Bucket(bucket).Object(data_key).upload_fileobj(buf)
    ```
    Run this file `python s3_sample_data_creator.py`
-3. Prepare an IAM role with permissions to run SageMaker jobs and access to S3 buckets. https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html  
-   Go to you AWS account -> IAM -> Roles -> Create Role -> Select SageMaker Service -> Next -> Next -> Next -> Give "SageMakerExecutorKFP" as Role Name -> Create Role -> Click on the role that you created -> Attach policy -> AmazonS3FullAccess -> Attach Policy -> Note down the role ARN
+3. Prepare an IAM role with permissions to run SageMaker jobs and access to S3 buckets. 
+   ```buildoutcfg
+   # create a new file trust.json with following content 
```

Comment:
Can we split the `trust.json` into its own separate step? This way we can have a block of code that users can copy and paste into their `trust.json`, and then they can have a block of code to copy and paste to create the role.

Reply:
Sometimes people see a big block of code formatted content as "I need to copy this into bash directly". This just helps make that a bit easier to differentiate.

Reply:
Done